docs/design/per-op-config.md

```diff
@@ -85,21 +87,71 @@ suspend fun main() {
 
 Some observations about the above sample:
 * Since no explicit HTTP engine is passed to construction of the root client, the SDK creates its own "managed" default.
-* The first use of `withConfig` and `use` demonstrates how to make basic one-off calls. The defaulted HTTP engine is
-  inherited by the child. When the service client is `closed()` as part of the `use { }` call, the runtime recognizes
+* The first use of `withConfig` and `use` demonstrates how to make basic one-off calls. The default HTTP engine is
+  shared with the new client. When the service client is closed as part of the `use { }` call, the runtime recognizes
   that others are still holding onto its HTTP engine and so it isn't fully closed.
-* The client passed to `doBackgroundRoutine` inherits the default HTTP engine as well. Both the parent and subclient can
-  be closed at any time in either order - the managed HTTP engine will only be reaped when both clients using it are
-  closed.
+* The client passed to `doBackgroundRoutine` shares the default HTTP engine as well. Both clients be closed at any time
+  in either order - the managed HTTP engine will only be closed when both clients using it are closed.
```

Comment:
**Style**: Separate independent clauses with an em dash (—) instead of a hyphen (-). No spaces surrounding the punctuation are necessary.